configs/FLYCOLORF7_AIO_HD/config.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        FLYCOLORF7_AIO_HD
+#define MANUFACTURER_ID   FLCO
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6500
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6500
+#define USE_MAX7456
+
+#define BEEPER_PIN PC13
+#define MOTOR1_PIN PC6
+#define MOTOR2_PIN PC7
+#define MOTOR3_PIN PB0
+#define MOTOR4_PIN PB1
+#define MOTOR5_PIN PB6
+#define MOTOR6_PIN PB8
+#define MOTOR7_PIN PB7
+#define MOTOR8_PIN PC9
+#define RX_PPM_PIN PA15
```

Comment:
Does this FC really have a PPM input pin?

Reply:
IDK, it even has a timer with dma slapped on it.